aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -556,6 +556,43 @@ void vdot<c10::complex<double>>(CUDABLAS_DOT_ARGTYPES(c10::complex<double>)) {
                                    reinterpret_cast<cuDoubleComplex*>(result)));
 }
 
+// This guards blocks use of getrfBatched and getriBatched on platforms other than cuda
+#ifdef CUDART_VERSION
+
+template <>
+void getrfBatched<double>(
+    int _m, int n, double** dA_array, int ldda, int* ipiv_array, int* info_array, int batchsize) {
```

Comment:
sorry, did not notice this earlier, but linter complains about unused param _m here (CUDABLAS_GETRF_ARGTYPES probably also need to be adjusted?)